internal/server/data/migrationdata/202206281027-postgres.sql

```diff
@@ -0,0 +1,902 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 14.2 (Debian 14.2-1.pgdg110+1)
+-- Dumped by pg_dump version 14.2 (Debian 14.2-1.pgdg110+1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: access_keys; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.access_keys (
```

Comment:
The postgres test drivers creates a schema `testing` and also sets the search_path to `testing`. I suspect this may not populate the database the way you're expecting

Reply:
As an aside, you can reduce the size of this file significantly by only exporting relevant tables
```
pg_dump --column-insert --table providers --table migrations
```

Reply:
You're right, this was not running the way I expected. Fixing it actually exposed some bugs in the code. Thanks!